Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    codeowners-checker (1.0.5)
+    codeowners-checker (1.1.0)
```

Comment:
Are you intent to bump here?

Reply:
I don't get why we don't push a minor version like `0.6` 🤔

Reply:
My reasoning behind it is because it introduces a deprecation, so it may require the user to run a command to configure its organization. So I went with a major since it requires a manual step. But I would be fine bumping it to 1.0.6 if its how the past releases behaved.

Reply:
got it now! sounds reasonable ✅